pkgs/applications/editors/spacevim/default.nix

```diff
@@ -0,0 +1,64 @@
+{ ripgrep, gitAndTools, fzf, makeWrapper, vim_configurable, vimPlugins, fetchFromGitHub, writeTextDir
+, stdenv, runCommandNoCC, remarshal, spacevim_config ? import ./init.nix }:
+with stdenv;
+let
+  # Once https://github.com/NixOS/nixpkgs/pull/75584 is merged we can use the TOML generator
+  toTOML = name: value: runCommandNoCC name {
+    nativeBuildInputs = [ remarshal ];
+    value = builtins.toJSON value;
+    passAsFile = [ "value" ];
+  } ''
+    mkdir $out
+    json2toml "$valuePath" "$out/${name}"
+  '';
+
+  vim-customized = vim_configurable.customize {
+    name = "vim";
+    # Not clear at the moment how to import plugins such that
+    # SpaceVim finds them and does not auto download them to
+    # ~/.cache/vimfiles/repos
+    vimrcConfig.packages.myVimPackage = with vimPlugins; { start = [ ]; };
+  };
+  spacevimdir = toTOML "init.toml" spacevim_config;
+in mkDerivation {
+  pname = "spacevim";
+  version = "unstable-2020-07-16";
+  src = fetchFromGitHub {
+    owner = "SpaceVim";
+    repo = "SpaceVim";
+    rev = "c937c0e2fd37207c36c8c5e53b36c41d7222fee6";
+    sha256 = "141fl5g6i2h72dk5121v3mc0bwb812hd8qa5qw83jyz9q20jcsgn";
+  };
+
+  nativeBuildInputs = [ makeWrapper vim-customized];
```

Comment:
```suggestion
  nativeBuildInputs = [ makeWrapper vim-customized ];
```

Reply:
You're really digging deep for the _nit_ feedback lol.

Reply:
when you read a few hundred expressions a day, it does become a particular eye sore. But a agreed that is a small nit.

Reviewing the whole PR will cost a lot of time, and i still have 200+ other github notifications to attend to :(

Reply:
I appreciate it; 

I would +1 just having a lint check against `nixfmt` or something one day :+1: